websites/A/Apple/metadata.json

```diff
@@ -90,4 +90,4 @@
 			"value": false
 		}
 	]
-}
+}
```

Comment:
Ensure you remove the changes made to the Apple and Stake metadata files, which were likely inadvertently changed when sorting your metadata. This is causing the "version bump" problems.

Reply:
I can't actually see what has actually been changed. It was a `yarn format`, could you advice what needs changing back

Reply:
If you take a look at the files tab of this PR, the metadata.json files for the apple and stake metadata files removed the EOF newline character.

Reply:
Fixed, sorry for the confusion 😅